### DISCUSSION TITLE
Track Apple Watch via BLE?

### DISCUSSION THREAD

**Discussion by hagak** [mKeRix/room-assistant#1048] [Q&A]
Sorry if this has been answered but my searches here and via google for this were not giving good results.

Anyhow, I have RA working tracking my Apple Watch via Bluetooth Classic and I have it tracking my iPhone via BLE using the companion app.  Is it possible to track the Watch via BLE?  If so any guidance?

**Comment by Ripper2909** [comment]
Maybe you can try to get it's BLE address via BT Inspector on the App Store

**Reply by hagak** [reply]
Tried this, used every UUID I could find on my watch.  No luck

**Reply by Ripper2909** [reply]
Damn... I am resetting my Apple Watch tomorrow so I was hoping this was the solution... Also looking for a solution here then 🙂

**Reply by hagak** [reply]
Hey maybe you have more luck with it, if you do please post details.

**Reply by Ripper2909** [reply]
https://github.com/mKeRix/room-assistant/issues/44 looks like there's a solution... Could you tell me if it solves your case ?

**Reply by hagak** [reply]
That gets it working with Bluetooth, not BLE.  Bluetooth is much slower to respond and much less accurate for distance measurement.  Also uses more power.  So all around kinda crappy for what I want to use my Apple Watch for.   It really is only good for saying if someone is home or not, not which room they are in since the resolution and response time is so poor.  For determine someone is home or not room-assistant is a bit overkill.

**Reply by Ripper2909** [reply]
Oh I see, I am new to this approach and I was not aware of that.
Hope we'll find a solution to use it with the BLE integration...
